ext/ewaysingle/lib/eWAY/eWAY_GatewayRequest.php

```diff
@@ -205,11 +205,54 @@ public function ToXml() {
    *
    */
   public function CreateNode($NodeName, $NodeValue) {
-    require_once E::path('lib/XML/Util.php');
-
-    $xml = new XML_Util();
-    $node = "<" . $NodeName . ">" . $xml->replaceEntities($NodeValue) . "</" . $NodeName . ">";
+    $node = "<" . $NodeName . ">" . $this->replaceEntities($NodeValue) . "</" . $NodeName . ">";
     return $node;
   }
 
+  /**
+   * replace XML entities
+   *
+   * With the optional second parameter, you may select, which
```

Comment:
So can we remove the optional, but unused, second parameter & the docblocks for this & the 3rd

Reply:
Updated now @eileenmcnaughton